R/inst/tutorials/01-playlist/README.md

```diff
@@ -11,8 +11,10 @@ in an R Markdown Notebook.**
 - Using the Metaflow client in an R Markdown Notebook.
 
 #### To play this episode:
-1. ```cd metaflow-tutorials/R/01-playlist/```
+1. ```cd tutorials/01-playlist/```
 2. ```Rscript playlist.R show```
 3. ```Rscript playlist.R run```
 4. ```Rscript playlist.R run --genre comedy```
-5. Open ```playlist.Rmd``` in RStudio.
\ No newline at end of file
+5. Open ```playlist.Rmd``` in RStudio.
+
+RStudio instructions. You can replace the last `run()` in `playlist.R` with `run(genre="comedy")`, and run the episode by `source("playlist.R")`.
```

Comment:
minor nit - _If you are using RStudio, you can replace the `run()` in last line in `playlist.R` with `run(genre="comedy")`, and run the episode by executing `source("playlist.R")`._

Reply:
updated according to your suggestion